src/errors.ts

```diff
@@ -76,3 +76,17 @@ export class MissingNewJobMessageError extends Error {
     Object.setPrototypeOf(this, MissingNewJobMessageError.prototype)
   }
 }
+
+export class InvalidHash extends Error {
+  constructor(hash: string) {
+    super(`Invalid SHA-1 commit hash: ${hash}`)
+    Object.setPrototypeOf(this, MissingQueueNameInCommitSubjectError.prototype)
+  }
+}
+
+export class InvalidShortHash extends Error {
```

Comment:
@yeraydavidrodriguez all errors have the suffix `Error`

Reply:
Fixed.